PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalPaymentIntent.kt

```diff
@@ -0,0 +1,29 @@
+package com.braintreepayments.api.paypal
+
+/**
+ * The payment intent in the PayPal Checkout flow
+ */
+enum class PayPalPaymentIntent(val stringValue: String) {
```

Comment:
Do we need to call this out in the changelog?

Reply:
Good catch! let me add this.